### DISCUSSION TITLE
How to remove the outline view?

### DISCUSSION THREAD

**Discussion by ZhiXiao-Lin** [eclipse-theia/theia#10542] [Q&A]
I want to remove the outline view from the editor.

I tried to delete @Theia/outline-view lib, but it didn't take effect.

I also tried rebind outlineview contribution and monacooutlinecontribution, but the outline is still there.

![image](https://user-images.githubusercontent.com/19965768/146178571-9cd53c53-9d5f-48cb-b59b-14c183778c6e.png)

I can't find another way. Can anyone help me?

**Comment by vince-fugnitto** [comment]
@ZhiXiao-Lin thank you for the discussion, your changes look good to me, can you confirm that the `outline-view` does not exist after performing a reset using the theia command (<kbd>F1</kbd> > `Reset Workbench Layout`).

**Reply by ZhiXiao-Lin** [reply]
> @ZhiXiao-Lin thank you for the discussion, your changes look good to me, can you confirm that the `outline-view` does not exist after performing a reset using the theia command (F1 > `Reset Workbench Layout`).

I tried to execute (F1 > Reset workbench layout), but the outline view is not hidden

**Reply by vince-fugnitto** [reply]
@ZhiXiao-Lin I'm not doing things much different than you, when I perform the reset the outline-view is gone, and it cannot be re-opened (not part of the view menu):

```ts
import { ContainerModule } from '@theia/core/shared/inversify';
import { OutlineViewContribution } from '@theia/outline-view/lib/browser/outline-view-contribution';

export default new ContainerModule((bind, unbind, isBound, rebind) => {
    rebind(OutlineViewContribution).toConstantValue({
        initializeLayout: () => { },
        registerCommands: () => { },
        registerKeybindings: () => { },
        registerMenus: () => { },
        registerToolbarItems: () => { }
    } as any)
});

```

```json
"dependencies": {
    "@theia/core": "latest",
    "@theia/editor": "latest",
    "@theia/filesystem": "latest",
    "@theia/markers": "latest",
    "@theia/messages": "latest",
    "@theia/monaco": "latest",
    "@theia/navigator": "latest",
    "@theia/outline-view": "latest",
    "@theia/preferences": "latest",
    "@theia/process": "latest",
    "@theia/terminal": "latest",
    "@theia/workspace": "latest",
    "no-outline": "0.0.0"
  },
  ```
